automation/jleechanorg_pr_automation/automation_safety_manager.py

```
@@ -750,6 +776,131 @@ def _is_email_configured(self) -> bool:
+        with self.lock:
+            # Reload PR override file to stay in sync with CLI updates from other processes.
+            overrides_data = self._read_json_file(self.pr_overrides_file)
+            self._pr_overrides_cache = {k: int(v) for k, v in overrides_data.items()}
+
+            pr_key = self._make_pr_key(pr_number, repo, branch)
+            override = self._pr_overrides_cache.get(pr_key)
+
+            if override is not None:
+                # 0 means unlimited
+                return sys.maxsize if override == 0 else override
+            else:
```

Ruff (0.14.11)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)